demo.py

```diff
@@ -1,3 +1,3 @@
 print('greetings')
 
-print('what do you think about it?')
\ No newline at end of file
+print('what do you think about this?????')
```

Comment:
not so many question marks please
```suggestion
print('what do you think about this?')
```

Reply:
nice